### DISCUSSION TITLE
[Support]: home assistnat add on breaks after upgrading to dev to fix /dev/shm filling and crashing issue

### DISCUSSION THREAD

**Discussion by zombiehoffa** [blakeblackshear/frigate#13963] [General Support]
### Describe the problem you are having

I switched from 0;14.1 to the latest dev commit docker image and now the hass addon is broken. I am getting mqtt connections but none of the entities are being provided. Is this just the expected behaviour since it's frigate dev, or can I do something here to get it working?

### Version

latest commit

### Frigate config file

```yaml
na.
```

### Relevant Frigate log output

```shell
I don't see anything obvious
```

### Relevant go2rtc log output

```shell
same deal
```

### FFprobe output from your camera

```shell
same deal
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

hikvision

**Comment by NickM-27** [comment]
this is a known issue

**Reply by zombiehoffa** [reply]
sad news because that means whatever I do my notification system via ntfy on home assistant is broken..

**Reply by NickM-27** [reply]
Why are you running dev builds if you want system stability? They are dev builds and done without releases for a reason

**Reply by zombiehoffa** [reply]
because the default build fills /dev/shm up to the max randomly and crashes the whole thing relatively frequently.

**Reply by NickM-27** [reply]
then I'd suggest finding a dev build that works and not continue to update to newer images. The latest PR has fixed the issue you were seeing

**Reply by zombiehoffa** [reply]
yes of course, but running that dev build breaks home assistant integration which breaks my notification pipeline, which is what the sad news was. My notification pipeline is broken either way.  either it breaks frequently and randomly because of /dev/shm or it is just broken because home assistant integration is broken. I think something was lost/misunderstood here, sorry, as I didn't expect this to be a 5 replies sort of comment.

**Reply by NickM-27** [reply]
> yes of course, but running that dev build breaks home assistant integration which breaks my notification pipeline

how do you know? It was only merged a couple minutes ago and the builds are still going

**Reply by zombiehoffa** [reply]
ahh, then yes the misunderstanding is on my part. You were saying home assistant integration is fixed in the latest commit, not that I should stick with an earlier unfixed for /dev/shm version, my apologies. English is my only language but I speak and understand it imperfectly.
